pages/examples/index.tsx

```diff
@@ -57,14 +58,15 @@ export async function getStaticProps() {
           getOptimizedImageAttributes(path.join(path.sep, dirInPublic, image)),
         ) || [],
       content: mdxSource,
+      weight: frontmatter.weight || 999999,
     });
   }
 
   const tags = examples.flatMap((example) => example.workflows).filter(Boolean);
 
   return {
     props: {
-      examples,
+      examples: examples.sort((a, b) => (a.weight - b.weight)),
```

Comment:
@markotoplak 
shouldn't the examples with higher weight be on top?
You are sorting from ascending